azure-pipelines.yml

```diff
@@ -475,6 +475,7 @@ jobs:
           --test_output=all \
           --build_tests_only \
           --define cw310=lowrisc \
+          --notest_keep_going \
```

Comment:
Does this change mean that if I make an RTL change that breaks 10 of the chip-level tests then I'll only be told about the first one? If so, I'm not super-keen. It's sometimes nice to get the list of things that a change has broken, since it often makes it obvious what you did! :-)

Reply:
There's another way I could try to keep the queue from filling up when tests start to hang and that's managing the timeouts a little better. I'll try that tomorrow.